requirements.txt

```diff
@@ -27,8 +27,8 @@ sentencepiece>=0.2.0
 # "old" version required for vLLM on CUDA to build
 tokenizers>=0.11.1
 toml>=0.10.2
-# Habana Labs 1.17.1 has PyTorch 2.3.1a0+gitxxx pre-release
-torch>=2.3.0,<2.4.0
+# Default version. Can be overridden in extra requirements
+torch>=2.4.1,<2.5.0
```

Comment:
Let's allow 2.3.x until the entire ecosystem has been moved to >=2.4.0. Also keep the comment about Intel Gaudi / Habana Labs. It's there to make sure that people do not break Intel Gaudi support by requiring an incompatible PyTorch version...

```suggestion
# Intel Gaudi software 1.17.1 has PyTorch 2.3.1a0+gitxxx pre-release
torch>=2.3.0,<2.5.0
```